client/js/test/table-view.test.js

```diff
@@ -11,6 +11,95 @@ describe('table-view', () => {
     document.documentElement.innerHTML = html;
   });
 
+
+
+  describe('add a row', () => {
+    it('should add a row', () => {
```

Comment:
What should? When? Make your test assertions descriptive of the functionality. A better one would be "if adds a row when the Add Row button is clicked", for example.